ImageFunctions/Thumbnail.cs

```diff
@@ -0,0 +1,117 @@
+// Default URL for triggering event grid function in the local environment.
+// http://localhost:7071/runtime/webhooks/EventGrid?functionName={functionname}
+
+// For local testing:
+// https://{ID}.ngrok.io/runtime/webhooks/EventGrid?functionName=Thumbnail
+
```

Comment:
Maybe add an aka.ms link to your new article in the comments?

Reply:
Great idea!